lambda-http/src/lib.rs

```diff
@@ -1,10 +1,10 @@
 #![warn(missing_docs)]
 //#![deny(warnings)]
-//! Enriches the `lambda_runtime` crate with [http](https://github.com/hyperium/http)
-//! types targeting ALB and API Gateway proxy events.
+//! Enriches the `lambda` crate with [http](https://github.com/hyperium/http)
+//! types targeting ALB, API Gateway REST and HTTTP API proxy events.
```

Comment:
```suggestion
//! types targeting ALB, API Gateway REST and HTTP API proxy events.
```

Note that I don't _think_ your changes handle the schema of the new HTTP APIs yet.

Reply:
getting there...

Reply:
Oops! My bad.